### PR TITLE
How to Build and Test a Node.js REST API

### PR DESCRIPTION
**Pull request by adrianmurage**

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋  Good afternoon @louisefindlay23  - wanted to ask if you could give this article a review when you have a moment. 
I've added the front matter and checked the links (while I was previewing it)
There are a few code blocks to verify. 
Let me know if you have any questions 🚀   
Thank you

**Comment by louisefindlay23** [comment]
> 👋 Good afternoon @louisefindlay23 - wanted to ask if you could give this article a review when you have a moment.
> I've added the front matter and checked the links (while I was previewing it)
> There are a few code blocks to verify.
> Let me know if you have any questions 🚀
> Thank you

@hectorkambow, thanks. I'm happy to tech review this article and have assigned myself. Will run through the code tomorrow and verify everything works as expected.

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (louisefindlay23) [inline articles/building-a-basic-api-with-nodejs/index.md:147]: Add comma after successful.

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (louisefindlay23) [inline articles/building-a-basic-api-with-nodejs/index.md:181]: Postman should be capitalised and a comma added afterwards.

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (louisefindlay23) [inline articles/building-a-basic-api-with-nodejs/index.md:207]: Capitalise Postman.

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (louisefindlay23) [inline articles/building-a-basic-api-with-nodejs/index.md:183]: While you do have an image that shows the example URL in Postman, it would be nice if you explained things more here especially for readers using screen readers who won't be able to see the screenshot. 

Something like "the endpoint is your ngrok URL followed by /list_movies."

**Review by louisefindlay23 — CHANGES_REQUESTED** [review]
Awesome article, @adrianmurage. I learned a lot about creating APIs with Node.js. You explained things well because I found it easy to follow depsite having never created an API before or worked with Ngrok or Postman.

There's just a few minor suggestions on how to improve syntax highlighting for code blocks, small grammar fixes and how to make the instructions more cross-platform and easy to understand.

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (adrianmurage) [reply articles/building-a-basic-api-with-nodejs/index.md:41]: Noted

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (adrianmurage) [reply articles/building-a-basic-api-with-nodejs/index.md:143]: I intended this article to guide users on Ubuntu 18.04 as denoted on the title.
I have no experience developing on windows hence I cannot guide one to do what I do not know myself

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (adrianmurage) [reply articles/building-a-basic-api-with-nodejs/index.md:183]: noted

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (adrianmurage) [reply articles/building-a-basic-api-with-nodejs/index.md:26]: That's a plus but I intend it to be an Ubuntu 18.04 guide that's clear enough to teach the concepts. So that one who wishes to develop on a different platform gets the core concepts then uses documentation to implement it on their specific platform.

**Comment by adrianmurage** [comment]
@hectorkambow @louisefindlay23 I made the changes requested. Kindly re-review.

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (louisefindlay23) [inline articles/building-a-basic-api-with-nodejs/index.md:36]: Sorry, I may have been slightly unclear regarding the proper markdown formatting for code. Single line code should only by surrounded by one backtick which you had before whereas multi-line code blocks (where you add js or json etc. for syntax highlighting) should be surrounded by three. https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

**Review by louisefindlay23 — APPROVED** [review]
Thanks, @adrianmurage for implementing the suggested changes. See my final comment before @hectorkambow merges. Looking forward to seeing it published.

File: articles/building-a-basic-api-with-nodejs/index.md
Comment (adrianmurage) [reply articles/building-a-basic-api-with-nodejs/index.md:36]: Thanks. I'll revert those changes.

**Comment by louisefindlay23** [comment]
Thanks, @adrianmurage. @hectorkambow, it's good to go.

**Comment by hectorkambow** [comment]
@ninjaginja  
Molly - if you could give this article a quick final review - Louise did a tech review and preliminary review - she tested and successfully built a Node.js API following the steps. 
The article seems well written - just wanted another pair of eyes (felt quite comfortable publishing this one)
Please let me know your thoughts 🚀 Thank you

**Pull request merged**
